src/m_pd.h

```diff
@@ -681,15 +691,15 @@ EXTERN int garray_getfloatwords(t_garray *x, int *size, t_word **vec);
 EXTERN void garray_redraw(t_garray *x);
 EXTERN int garray_npoints(t_garray *x);
 EXTERN char *garray_vec(t_garray *x);
-EXTERN void garray_resize(t_garray *x, t_floatarg f);  /* avoid; use this: */
+PDDEPRECATED EXTERN void garray_resize(t_garray *x, t_floatarg f); /* use garray_resize_long() */
 EXTERN void garray_resize_long(t_garray *x, long n);   /* better version */
 EXTERN void garray_usedindsp(t_garray *x);
 EXTERN void garray_setsaveit(t_garray *x, int saveit);
 EXTERN t_glist *garray_getglist(t_garray *x);
 EXTERN t_array *garray_getarray(t_garray *x);
 EXTERN t_class *scalar_class;
 
-EXTERN t_float *value_get(t_symbol *s);
+PDDEPRECATED EXTERN t_float *value_get(t_symbol *s); /* use value_getfloat() */
```

Comment:
As noted in https://github.com/pure-data/pure-data/pull/817#issuecomment-563009409, `value_getfloat` is not a simple replacement of `value_get`. While the former just returns the current float value and returns false if the value object doesn't exist, the latter creates a strong reference to the value object (which has to be released with `value_release`) and also creates the value object itself if needed. Until we have a better replacement, like my proposed `value_retain` method, we shouldn't deprecate it.

Reply:
i would recommend setting the deprecated flag in multiple steps:
- first mark functions as deprecated where we *know* that they are indeed deprecated because they are already marked as deprecated in the headers (e.g. `clock_getsystime`, `garray_resize`)
- in a separate commit mark functions that are deprecated that say so when being called (e.g. `garray_getfloatarray`)
- finally mark functions that are supposedly deprecated (~`value_get`~, if you really think it is)